src/sample-data/aml-check-private-key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Shouldn't these files be in the tests folder?

Reply:
Most tests files are inside the `src/sample-data` - this file can be moved inside the `tests` folder and will have to move all files that are only used in `tests` only.